js/components/modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { Modal as RestartModal } from "@restart/ui";
+import { ReactElement } from "react";
+
+export const Modal = ({
+  children,
+  title,
+  show,
+  onHide,
+}: {
+  children: ReactElement;
+  title: ReactElement;
+  show: boolean;
+  onHide: () => void;
+}): ReactElement => {
+  return (
+    <RestartModal
+      show={show}
+      onHide={onHide}
+      className="fixed z-modal-content flex max-h-[calc(100vh-3.5rem)] flex-col bg-gray-100 border-2 border-black p-1 -translate-x-2/4 -translate-y-2/4 top-2/4 left-2/4"
+      renderBackdrop={(props) => (
+        <div
+          {...props}
+          className="fixed inset-0 z-modal-backdrop w-full h-full bg-black/[0.48]"
+        />
+      )}
+    >
+      <div className='grid grid-cols-[1fr_max-content] grid-rows-[max-content_1fr] [grid-template-areas:"title_close"_"contents_contents"]'>
+        <div className="[grid-area:title]">{title}</div>
+        <button className="[grid-area:close]" onClick={onHide}>
+          [x]
```

Comment:
I'm partial to no square brackets, but 🤷

Reply:
Yeah, my preference is that we just get the icons working eventually and then ditch this version entirely.